examples/batch.R

```diff
@@ -0,0 +1,91 @@
+# Batch examples
+
+# To set up the Batch compute environment, get security group and subnet info
+# for the default VPC.
+default_vpc <- paws.ec2::describe_vpcs(
+  Filters = "isDefault=true"
+)$Vpcs[[1]]
+
+security_group <- paws.ec2::describe_security_groups(
+  Filters = sprintf("vpc-id=%s", default_vpc$VpcId),
+  GroupNames = "default"
+)$SecurityGroups[[1]]
+
+subnets <- paws.ec2::describe_subnets(
+  Filters = sprintf("vpc-id=%s", default_vpc$VpcId)
+)$Subnets
+
+# You will need to set up the Batch service role and provide your AWS account
+# account number here.
+service_role <- sprintf(
+  "arn:aws:iam::%s:role/service-role/AWSBatchServiceRole",
+  account_id = "1234567890123"
+)
+
```

Comment:
The service role shouldn't have the "/service-role" section in the middle. This should be:
```
service_role <- sprintf(
  "arn:aws:iam::%s:role/AWSBatchServiceRole",
  account_id = "1234567890123"
)
```

Reply:
@davidkretch I don't think this has been addressed yet

Reply:
This is no longer in the PR. It now sets up a new IAM role instead.